Makefile

```diff
@@ -54,7 +58,6 @@ minify: dist/rdflib.min.js
 writable:
 	@sed -i -re 's/git:\/\/github.com\//git@github.com:/' .git/config
 
-.PHONY: test
```

Comment:
I haven't written Makefiles in a while, so I dunno. I like your suggestion though. I'll add a `.PHONY` on the line prior to each phony target.